src/main/java/com/example/payroll/Controllers/EmployeesController.java

```diff
@@ -0,0 +1,43 @@
+package com.example.payroll.Controllers;
+
+import com.example.payroll.Models.EmployeesDTO;
+import com.example.payroll.Repositories.EmployeesRepository;
+import com.example.payroll.Services.EmployeesService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class EmployeesController {
+
+    @Autowired
```

Comment:
There are different ways to wire a class in a bean.
I suggest making these dependencies `final`, adding `@RequiredArgsConstructor` and removing `@Autowired`

Reply:
Done.